ios/Panthalassa.m

```diff
@@ -276,19 +276,22 @@ - (dispatch_queue_t)methodQueue
                  PanthalassaDecryptMessageWithResolver:(NSDictionary *)config
                  resolver:(RCTPromiseResolveBlock)resolve
                  rejecter:(RCTPromiseRejectBlock)reject) {
+  dispatch_queue_t queue = dispatch_queue_create("panthalassaLibQueueNew", DISPATCH_QUEUE_SERIAL);
```

Comment:
Do we actually have this changes? I don't remember that

Reply:
We changed that later... guess on 0.5.3. Not sure really

Reply:
Actually I see it in 0.5.3 as well.. I guess it's not a bug, just the rest of our first try of fixing thread blocking issue. We can remove it in a 0.5.4 release then.